Added a bit of text to the 'Supported product versions' page (#569)
stackabletech/documentation: modules/operators/pages/supported_versions.adoc

= Supported product versions

Below you find the list of versions that are supported for each product.

> New versions get added every release (and old ones deprecated), read the xref:ROOT:release-notes.adoc[] for each version to see which versions have been added or removed.

Reviewer: ```suggestion
New versions get added every release, old ones are deprecated or removed. Read the xref:ROOT:release-notes.adoc[] for each platform version to see which product versions are available.
```
